dev-helpers/template.ejs

```diff
@@ -5,7 +5,6 @@
   <meta charset="UTF-8">
```

Comment:
`dev-helpers/index.html` needs to be intact, since the hot dev server uses it - unless you've configured the dev server to use this template as well.